pyspark/test/dev/prepare_env.sh

```diff
@@ -33,7 +33,7 @@ export PYTHONPATH=$PYTHONPATH:$PYSPARK_ZIP:$DL_PYTHON_HOME:$DL_PYTHON_HOME/:$DL_
 export BIGDL_CLASSPATH=$(find $BIGDL_HOME/spark/dl/target/ -name "*with-dependencies.jar" | head -n 1)
 echo "BIGDL_CLASSPATH": $BIGDL_CLASSPATH
 
-export PYTHON_EXECUTABLES=("python2.7" "python3.5")
+export PYTHON_EXECUTABLES=("python2.7" "python3.6")
```

Comment:
need to keep both python3.5 and python3.6

Reply:
Yeah. This is only for test purpose. Will add it later.